src/main/java/org/folio/exceptions/FailedToCreateRepositoryException.java

```diff
@@ -0,0 +1,7 @@
+package org.folio.exceptions;
+
+public class FailedToCreateRepositoryException extends RuntimeException {
```

Comment:
Failed to create a repository?

Reply:
Sonar asked to use a specific exception instead of generic one

Reply:
I mean, the name suggests that this exception is thrown when we fail to create (instantiate?) a repostitory. Is that how we use it?